src/test/java/bio/terra/integration/DataRepoFixtures.java

```diff
@@ -626,6 +627,33 @@ public BulkLoadArrayResultModel bulkLoadArray(
     }
   }
 
+  public BulkLoadHistoryModelList getLoadHistory(
+      TestConfiguration.User user, UUID datasetId, String loadTag, int offset, int limit)
+      throws Exception {
+    var response =
+        dataRepoClient.get(
+            user,
+            "/api/repository/v1/datasets/"
+                + datasetId
+                + "/files/bulk/"
+                + loadTag
+                + "?offset="
+                + offset
+                + "&limit="
+                + limit,
+            BulkLoadHistoryModelList.class);
+    if (response.getStatusCode().is2xxSuccessful()) {
+      assertThat("getLoadHistory is successful", response.getStatusCode(), equalTo(HttpStatus.OK));
+      assertTrue("getLoadHistory response is present", response.getResponseObject().isPresent());
+      return response.getResponseObject().get();
+    } else {
+      ErrorModel errorModel = response.getErrorObject().orElse(null);
+      logger.error("getLoadHistory failed: " + errorModel);
+      fail();
+      return null; // Make findbugs happy
```

Comment:
Is this better than?
```suggestion
      throw new AssertionError();
```

If you throw than findbugs shouldn't complain.